owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/utils/Log_OC.java

```diff
@@ -1,212 +1,52 @@
 package com.owncloud.android.lib.common.utils;
 
-import android.util.Log;
+import timber.log.Timber;
 
-import com.owncloud.android.lib.BuildConfig;
-
-import java.io.BufferedWriter;
 import java.io.File;
-import java.io.FileWriter;
-import java.io.IOException;
-import java.text.SimpleDateFormat;
-import java.util.Calendar;
-import java.util.Locale;
 
 public class Log_OC {
-    private static final String SIMPLE_DATE_FORMAT = "yyyy/MM/dd HH:mm:ss";
-    private static final String LOG_FOLDER_NAME = "log";
-    private static final long MAX_FILE_SIZE = 2000000; // 2MB
-
-    private static String mOwncloudDataFolderLog = "owncloud_log";
-
-    private static File mLogFile;
-    private static File mFolder;
-    private static BufferedWriter mBuf;
-
-    private static String[] mLogFileNames = {
-            "currentLog" + BuildConfig.BUILD_TYPE + ".txt",
-            "olderLog" + BuildConfig.BUILD_TYPE + ".txt"
-    };
 
-    private static boolean isMaxFileSizeReached = false;
-    private static boolean isEnabled = false;
+    private static String mOwncloudDataFolderLog;
 
     public static void setLogDataFolder(String logFolder) {
         mOwncloudDataFolderLog = logFolder;
     }
 
-    public static void i(String TAG, String message) {
-        Log.i(TAG, message);
-        appendLog("I: " + TAG + " : " + message);
+    public static void i(String tag, String message) {
+        Timber.i(message);
     }
 
     public static void d(String TAG, String message) {
-        Log.d(TAG, message);
-        appendLog("D: " + TAG + " : " + message);
+        Timber.d(message);
     }
 
     public static void d(String TAG, String message, Exception e) {
-        Log.d(TAG, message, e);
-        appendLog("D: " + TAG + " : " + message + " Exception : " + e.getStackTrace());
+        Timber.d(e, message);
     }
 
     public static void e(String TAG, String message) {
-        Log.e(TAG, message);
-        appendLog("E: " + TAG + " : " + message);
+        Timber.d(message);
```

Comment:
To keep same format, could we use Timber.e(message) ?

Reply:
You are right, good catch